cli/command/container/list.go

```diff
@@ -40,6 +40,9 @@ func NewPsCommand(dockerCli command.Cli) *cobra.Command {
 		Annotations: map[string]string{
 			"category-top": "3",
 		},
+		ValidArgsFunction: func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
+			return nil, cobra.ShellCompDirectiveError
```

Comment:
Should this use `cobra.ShellCompDirectiveNoFileComp` ? Looks like currently it falls back to default (complete filenames); changing it to `ShellCompDirectiveNoFileComp` disables completion (probably same for some others; also see my comment on considering to have a reusable function for this perhaps 🤔 .

```suggestion
			return nil, cobra.ShellCompDirectiveNoFileComp
```